assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterAnalyticsAddresser.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2012-2016 Codenvy, S.A.
```

Comment:
Well, depending on the upstream `war` for overlays doesn't bring transitive dependencies.
see [this comment](https://github.com/redhat-developer/rh-che/pull/32/files#r111462819)